sensor_msgs_py/test/test_flake8.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Change this to the BSD license to match the declared license of the package.

Reply:
In `point_cloud2.py` the the copyright is given as `Copyright (c) 2008, Willow Garage, Inc.`
Should this be used here as well? Both? 

And what about the year, should that be updated?

Reply:
> In `point_cloud2.py` the the copyright is given as `Copyright (c) 2008, Willow Garage, Inc.`
> Should this be used here as well? Both?

This should be changed to be `Copyright (c) 2020 Open Source Robotics Foundation, Inc.`.  The reasoning here is that this is derived from other OSRF code, so it makes sense to have this copyright be OSRF as well.

> And what about the year, should that be updated?

Yeah, the year should be updated to 2020.